src/styles/abstracts/_vars-utility.scss

```diff
@@ -3,56 +3,28 @@
 $text-color-primary: $neutral-00;
 $text-color-ghost: $neutral-100;
 
-// Border Utility Variables
-
-$border-color-primary: $neutral-00;
-$border-color-ghost: $neutral-100;
-$border-color-accent: $action-primary-40;
-$border-color-subtle: $neutral-80;
-
-@mixin border-gradient-accent {
-  border-width: space(sm);
-  border-style: solid;
-  border-image-source: $gradient-accent;
-  border-image-slice: 1;
-}
-
-$border-radius: 0.25rem; // NOTE: this is a one-off size
-
-// Shadow Utility Variables
-
-$box-shadow-primary: 0 0 0.75rem 0 rgba($neutral-00, 0.2); // NOTE: this is a one-off size
-$box-shadow-secondary: 0 0.125rem 1rem 0 rgba($neutral-00, 0.1);
-$box-shadow-tertiary: 0 0.125rem 0.625rem 0 rgba($neutral-00, 0.2);
-
 // Breakpoint Utility Variables
 // TODO: ADS-197 set up mixins for media queries
```

Comment:
Did we keep this one on purpose? ADS-197 maps to some weird educare story now, so we'll probably want to sort that out later